pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/rbac/kustomization.go

```diff
@@ -60,4 +60,5 @@ const kustomizeRBACTemplate = `resources:
 - auth_proxy_role.yaml
 - auth_proxy_role_binding.yaml
 - auth_proxy_client_clusterrole.yaml
+# Editor and Viewer roles for each CRD to be used by end users.
```

Comment:
Lets just add this line when we create/scaffold an API and not via the init by default.
WDYT?

Reply:
Agree, it would be more clear.

Do we still need to add a marker (such as `#+kubebuilder:scaffold:..`) as the InsertCode's target in this kustomization file? Or is there any better approach? I'd appreciate it if you could shed some more light on this!

Reply:
No, just insert this line when we create the first API. Because today we create the file when we run kb init. So, if nobody run create API project than would not make sense have the comment without any API/CRD bellow right?

If you think that is to hard and get stuck that is fine we can move with it as it is.